depthai_ros_driver/config/rgbd.yaml

```diff
@@ -0,0 +1,49 @@
+/**:
+  i_pipeline_type: "RGBDM"
+  i_enable_imu: True
+  i_usb_speed: "SUPER"
+  i_cam_type: "auto"
+  i_mxid: ""
+  i_ip: ""
+  RGB.i_board_socket_id: 0
+  RGB.i_publish_preview: False
+  RGB.i_fps: 30.0
+  RGB.i_preview_size: 256
+  RGB.i_rgb_width: 1920
+  RGB.i_rgb_heigh: 1080
+  RGB.i_resolution: "1080"
+  RGB.i_max_q_size: 4
+  RGB.i_set_isp: False
+  RGB.i_interleaved: False
+  RGB.i_keep_preview_aspect_ratio: True
+  RGB.r_exposure: 1000
+  RGB.r_iso: 100
+  RGB.r_focus: 1
+  RGB.r_whitebalance: 3300
+  RGB.r_set_man_focus: True
+  RGB.r_set_man_exposure: False
+  RGB.r_set_man_whitebalance: False
+  MONO.i_fps: 30.0
+  MONO.i_resolution: "720"
+  MONO.r_iso: 800
+  MONO.r_set_man_exposure: False
+  MONO.r_exposure: 20000
+  STEREO.i_input: "MONO"
+  STEREO.i_lr_check: False
+  STEREO.i_align_depth: True
+  STEREO.i_default_profile_preset: "HIGH_DENSITY"
+  STEREO.i_lrc_threshold: 5
+  STEREO.i_depth_filter_size: 7
+  STEREO.i_stereo_conf_threshold: 255
+  STEREO.i_subpixel: True
+  STEREO.i_extended_disp: False
+  STEREO.i_rectify_edge_fill_color: -1
+  STEREO.i_enable_speckle_filter: False
+  STEREO.i_speckle_range: 5-
+  STEREO.i_enable_temporal_filter: True
+  STEREO.i_enable_spatial_filter: True
+  STEREO.i_hole_filling_radius: 2
+  STEREO.i_spatial_filter_iterations: 1
+  STEREO.i_spatial_filter_min_range: 400
+  STEREO.i_spatial_filter_max_range: 15000
+  STEREO.i_decimation_factor: 1
```

Comment:
Additional to this. I would add 
`enableDistortionCorrection` option for users who want do undistort even on NFOV
and `setOutputDepth` for someone who want to use even lower resolution like 100x200 something for approx example. Had a user in discord request this a while ago. 

https://docs.luxonis.com/projects/api/en/latest/references/python/#depthai.node.StereoDepth.setOutputDepth

Reply:
added `enableDistortionCorrection`,`setOutputDepth` seems deprecated